### PR TITLE
Postgres terminology change

### PR DESCRIPTION
**Pull request by kav**
Bringing this one back to life since some branch deletes killed the previously merged copy here https://github.com/bitnami/charts/pull/3079. Might need a major version branch/rev

**Description of the change**
Replaces master and slave with primary and readReplicas respectively. I've coalesced the respective values so existing configurations will continue to work but have updated all the docs to point to the newer terms.

**Benefits**
Let's use more inclusive language. Honestly it's more clear with primary and read replica terms what the actual difference is. Happy to tweak further of course. Just saw in my comment on the linked issue it perhaps should be standby?

**Possible drawbacks**
We currently use master and slave in pod names and labels and I've transitioned that. This may break custom scripts that are looking for those markers. I've bumped as non-breaking since values will still work fine but this might justify amending that.

I've not changed environment variables or anything that may require work on the image itself. Yet.

**Applicable issues**

  - related https://github.com/bitnami/bitnami-docker-postgresql/issues/175

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for creating again the PR, please, can you resolve the existing conflicts? I will mark this PR as `on-hold` to prevent it from closing automatically, then when there are more changes that require a major bump we'll include everything together to avoid different majors breaking the backward compatibility

**Comment by kav** [comment]
@carrodher this is likely to continuously have merge conflicts every time something else gets merged. It involves minor changes all over the place.  Any chance we can come up with a strategy to get it in so I don't have to constantly update?

**Comment by andresbono** [comment]
Hi @kav, can you  allow "edits from maintainers"? That way, we can resolve the conflicts ourselves once we are ready to merge the PR. Find more information in the link below:
- https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

It that option is not possible, you can leave the conflicts there until we ping you again.

**Comment by kav** [comment]
It's not a personal fork so I'm hitting the same issue @git-lucky did. but I added you both individually for now. Let me know if I should add anyone else specifically.

**Comment by andresbono** [comment]
Thanks @kav

**Comment by carrodher** [comment]
Hi @kav, we are going to do a major change in PostgreSQL, I just created this PR doing the required changes: https://github.com/bitnami/charts/pull/4293

As we talked in the past, in order to avoid different major version bumps, we can incorporate those changes to your PR and I will discard mine.

The required changes can be done automatically by executing the following commands:
```console
$ cat requirements.yaml >> Chart.yaml
$ rm -rf requirements.*
$ sed -i "s@version: 0.x.x@version: 1.x.x@g" Chart.yaml
$ sed -i "s@apiVersion: v1@apiVersion: v2@g" Chart.yaml
$ yq r --tojson Chart.yaml | jq . | yq r --prettyPrint - > Chart.yaml.new
$ mv Chart.yaml.new Chart.yaml
$ sed -i "s@Helm 2.12+ or Helm 3.0-beta3+@Helm 3.0-beta3+@g" README.md
```
then you should replicate the changes into the "Upgrade" section. 

Once done, this PR will contain your terminology changes and the changes related to the `apiVersion`.

**Comment by kav** [comment]
Perfect, think I got them all though my sed and yq are of course slightly different variants so did a bit of tweaking to the script. Output should match though. `python-yq` forever!

File: bitnami/postgresql/README.md [hunk 1]
```diff
@@ -520,7 +520,33 @@ $ helm upgrade my-release stable/postgresql \
...
+- If you want to upgrade to this version using Helm v2, this scenario is not supported as this version doesn't support Helm v2 anymore
+- If you installed the previous version with Helm v2 and wants to upgrade to this version with Helm v3, please refer to the [official Helm documentation](https://helm.sh/docs/topics/v2_v3_migration/#migration-use-cases) about migrating from Helm v2 to v3
+
+**Useful links**
+
+- https://docs.bitnami.com/tutorials/resolve-helm2-helm3-post-migration-issues/
+- https://helm.sh/docs/topics/v2_v3_migration/
+- https://helm.sh/blog/migrate-from-helm-v2-to-helm-v3/
+
+#### Breaking changes
```
Comment (carrodher) [inline bitnami/postgresql/README.md:547]: Here, in the "Breaking changes" subsection inside "To 10.0.0" we should mention the terminology renaming

**Review by carrodher — COMMENTED** [review]
Thanks for applying for the changes! I just put a comment in the README. I think apart from that (once resolved the merge conflicts) we can go ahead

**Comment by miguelaeh** [comment]
Hi,
We are merging this PR into the above one since we need to incorporate this changes into the new Major version.
Regards.

**Pull request closed without merging**